CodeEdit/Features/Documents/Controllers/CodeEditWindowController.swift

```diff
@@ -79,13 +80,18 @@ final class CodeEditWindowController: NSWindowController, NSToolbarDelegate, Obs
         let feedbackPerformer = NSHapticFeedbackManager.defaultPerformer
         let splitVC = CodeEditSplitViewController(workspace: workspace, feedbackPerformer: feedbackPerformer)
 
-        let navigatorView = SettingsInjector {
-            NavigatorSidebarView(workspace: workspace)
+        let navigatorViewModel = NavigatorSidebarViewModel()
```

Comment:
Here a `NavigatorSidebarViewModel` is also created, but just to pass it onto the `NavigatorSidebarView`. It's not used in other places in this class

Reply:
Because `navigatorSidebarViewModel` is optional, I cant pass it into `NavigatorSidebarView` without force unwrapping it, so I create the model in a variable so I can pass it into `NavigatorSidebarView`